src/pybind/mgr/nfs/export.py

```diff
@@ -637,6 +660,13 @@ def create_cephfs_export(self,
                              access_type: str,
                              clients: list = [],
                              sectype: Optional[List[str]] = None) -> Tuple[int, str, str]:
+        fs_client = ExportClient(self.mgr)
+        with open_fs_client(fs_client, fs_name) as fs_handle:
```

Comment:
can't this be:

```
from mgr_util import CephfsClient, open_filesystem
...
with open_filesystem(self, fs_name) as fs_handle:
    stat check
```

?

Reply:
oh, well maybe, then no need of creating child class it seems? need to test it

Reply:
updated, PTAL

Reply:
This would not be possible as this instance will be of type `ExportMgr` while open_filesystem needs `CephfsClient`, will add a new class with `__init__` overridden.

Reply:
created an instance of `CephfsClient` in `__init__` which is used in `open_filesystem()`